chsdi/templates/htmlpopup/lubis.mako

```diff
@@ -219,7 +219,7 @@ viewer_url = get_viewer_url(request, params)
         <td>${filename or '-'}</td></tr>
 % endif
 
-% if c['attributes'].get('doi_link', '').startswith('http'):
+% if c['attributes'].get('doi_link', '') or '-'.startswith('http'):
```

Comment:
for what is the second argument in the .get function? if it would be a fallback value, we could put '-' there...?

Reply:
it will result in False if the attribute does not exist:
```python
 c['attributes'].get('doi_link', '-') or '-'.startswith('http')
False
```

Reply:
if the get('doi_link', '-') results in None we would end up with this error:
```python
None.startswith('http')
exec("""exec code in globs, locs""")
File "<string>", line 1, in <module>
File "<debugger>", line 1, in <module>
None.startswith('http')
```

Reply:
`'-'.startswith('http')` will always be false and somewhat meaningless. better cast to str.